NEWS

```diff
@@ -91,6 +91,10 @@ CHANGES WITH 240 in spe:
           Consult the kernel documetnation for details on this sysctl:
           https://www.kernel.org/doc/Documentation/networking/ip-sysctl.txt
 
+        * CPUAccounting=yes no longer enables the CPU controller when using
+          kernel 4.15+ and the unified cgroup hierarchy, as required accounting
+          statistics are now provided externally from the CPU controller.
```

Comment:
not a native English speaker myself, but wouldn't "independently" instead of "externally" fit better here?

Reply:
Yes, that's much better. Thanks!